.github/actions/deploy-hyp3/action.yml

```diff
@@ -38,8 +38,10 @@ inputs:
   DEFAULT_CREDITS_PER_USER:
     description: "The default number of credits given to a new user"
     required: true
+  RESET_CREDITS_MONTHLY:
```

Comment:
`RESET_CREDITS_EACH_MONTH` might be a little more expressive

Reply:
It occurs to me that resetting could be driven by `DEFAULT_CREDITS_PER_USER != 0`. Thoughts on combining them into a single parameter? I kind of like the separate explicit parameter